src/lib/viewers/controls/media/VolumeControls.tsx

```diff
@@ -39,13 +38,24 @@ export default function VolumeControls({ onMuteChange, onVolumeChange, volume =
 
     const handleVolume = React.useCallback(
         (newValue: number): void => {
-            onVolumeChange(newValue / 100);
+            const newValueToUse = newValue <= 5 ? 0 : newValue;
```

Comment:
Why are we making values 5 or less 0?

Reply:
Because we don't have a track ball like we do with the horizontal slider so when the user pulls the slider to the bottom of the track we need to keep some height so that they can later increase the volume. That's where 5 comes in.